packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Combination/BasicCombinationGraphs/BarGraphWithLine.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import CombinationGraph from '@cerner/terra-graphs/lib/components/Combination/CombinationGraph';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import getCombinationGraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault';
+import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/BarGraphWithLineData';
+
+const graphConfig = utils.deepClone(getCombinationGraphConfig('#barGraphWithLine'));
+graphConfig.axis.x.ticks = {
+  values: [1, 2, 3, 4, 5, 6, 7],
+  format: '.0f',
+};
+graphConfig.showVGrid = false;
+graphConfig.axis.y2.show = true;
+graphConfig.axis.y2.padDomain = false;
```

Comment:
This config has been modified in the same way multiple times. It's better to make a new config object for it.

Reply:
Removed [here](https://github.com/cerner/terra-graphs/pull/150/commits/768d7923e4f325eaef40c8caa6ff6753069fc96e#diff-27e78132cd63323c97f901e43899c2b09ebcba76ab8039b2a796be137942ed78L9).